testnet-croeseid-2/README.md

```diff
@@ -2,8 +2,173 @@
 
 - chain-maind version: [v0.8.0-rc1+](https://github.com/crypto-com/chain-main/releases)
 - Seeds:
+
 ```
 b2c6657096aa30c5fafa5bd8ced48ea8dbd2b003@52.76.189.200:26656
 ef472367307808b242a0d3f662d802431ed23063@175.41.186.255:26656
 d3d2139a61c2a841545e78ff0e0cd03094a5197d@18.136.230.70:26656
-```
\ No newline at end of file
+```
+
+# Network upgrade guide:
+
+This is a guide for existing validator to upgrade from `testnet-croeseid-1` (v0.7._) to `testnet-croeseid-2` (v0.8._): 
+
+To simply the following steps, we will be using **Linux** for illustration. Binary for
+[Mac](https://github.com/crypto-com/chain-main/releases/download/v0.8.0-rc1/chain-main_0.8.0-rc1_Darwin_x86_64.tar.gz) and [Windows](https://github.com/crypto-com/chain-main/releases/download/v0.8.0-rc1/chain-main_0.8.0-rc1_Windows_x86_64.zip) are also available.
+
+### Step 1
+Stop the `chain-maind` and download the released binaries from github:
+
+  ```bash
+  $ curl -LOJ https://github.com/crypto-com/chain-main/releases/download/v0.8.0-rc1/chain-main_0.8.0-rc1_Linux_x86_64.tar.gz
+  $ tar -zxvf chain-main_0.8.0-rc1_Linux_x86_64.tar.gz
+  ```
+
+Remarks: If you have stated `chain-maind` with  systemd service, remember to stop it by  `sudo systemctl stop chain-maind`
+### Step 2
+Using the new binary, remove the old blockchain data by running
+
+```bash
+$ ./chain-maind unsafe-reset-all
+
+INF Removed existing address book   file=/Users/.chain-maind/config/addrbook.json
+INF Removed all blockchain history  dir=/Users/.chain-maind/data
+INF Reset private validator file to genesis state keyFile=/Users/.chain-maind/config/priv_validator_key.json stateFile=/Users/.chain-maind/data/priv_validator_state.json
+```
+
+
+### Step 3 Configure `chain-maind`
+
+- Download the and replace the Croseid Testnet `genesis.json` by:
+
+  ```bash
+  $ curl https://raw.githubusercontent.com/crypto-com/testnets/main/testnet-croeseid-2/genesis.json > ~/.chain-maind/config/genesis.json
+  ```
+
+- Verify sha256sum checksum of the downloaded `genesis.json`. You should see `OK!` if the sha256sum checksum matches.
+
+  ```bash
+  $ if [[ $(sha256sum ~/.chain-maind/config/genesis.json | awk '{print $1}') = "af7c9828806da4945b1b41d434711ca233c89aedb5030cf8d9ce2d7cd46a948e" ]]; then echo "OK"; else echo "MISMATCHED"; fi;
+
+  OK!
+  ```
+
+- For Cosmos configuration, in `~/.chain-maind/config/app.toml`, update minimum gas price to avoid [transaction spamming](https://github.com/cosmos/cosmos-sdk/issues/4527)
+
+  ```bash
+  $ sed -i.bak -E 's#^(minimum-gas-prices[[:space:]]+=[[:space:]]+)""$#\1"0.025basetcro"#' ~/.chain-maind/config/app.toml
```

Comment:
For a upgrade user, he/she should already done this.

Reply:
That's right 👍